html-report/src/components/LogContainer.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import LogFilter from './LogFilter'
+
+export default class LogContainer extends Component {
+
+  state = {
+    logs: [],
+    results: [],
+    loading: true,
+    hide: false
+  }
+
+  componentDidMount() {
+    document.addEventListener('DOMContentLoaded', (event) => {
+      this.loadStaticLogs()
+    })
+  }
+
+  loadStaticLogs() {
+    let logDivs = document.querySelectorAll('#static_logs > div > div')
+    if (!logDivs.length) {
+      this.setState({hide: true, loading: false})
+      return
+    }
+    let logs = []
+    for (var div of logDivs) {
+      logs.push(Object.assign({level: div.className, line: div.innerText, key: logs.length}))
```

Comment:
There's no sense in using `Object.assign` here — what you actually want is just to add a new object to the array. `Object.assign` is a method used to perform more complicated actions with the objects. Please see the documentation here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Also, if you want to add a unique key, here comes the `map` method — it has a unique index for each item in the array.